common/http.go

```diff
@@ -60,6 +60,17 @@ func (apiError APIError) Error() string {
 	return apiError.Message
 }
 
+// IsMissing tells if error is about missing resource
+func IsMissing(err error) bool {
+	if err == nil {
+		return false
+	}
+	if e, ok := err.(APIError); ok && e.IsMissing() {
```

Comment:
Simplify it to the 

e, ok := ….
return ok && e.IsMissing()

?

Reply:
Good one